.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**Type of PR**: Bugfix / Feature / Docs / Other
+
+**Required reviews**: 1 / 2 / 3
```

Comment:
I've seen some templates that use optional checkboxes for these kinds of questions - see https://github.com/appium/appium/blob/master/.github/PULL_REQUEST_TEMPLATE.md. Could be something to think about - I like them, but that's just me.

Reply:
Is there someway to hook the checkboxes into the automated Github enforcement of approval requirements?

Reply:
I wouldn't mind changing it to checkboxes, although I have a not great experience with them. Sometimes, GH arbitrarily renders them as `[ ]` or `[X]` (in ascii) instead of a nice checkboxes. Also I wanted to have the template as small as possible to fit as much as possible in the default textbox when editing the PR description. In any case, it's not a strong opinion on my side.

Reply:
> Is there someway to hook the checkboxes into the automated Github enforcement of approval requirements?

Unsure, but that would be cool. Our current one is a github branch rule for merging into main.

> Sometimes, GH arbitrarily renders them as [ ] or [X] (in ascii) instead of nice, real checkboxes

I think once we use `- [ ]` it should be fine - haven't had problems with that in PR/Issue description text - I don't think it works in comments.

>  I wanted to have the template small to fit as much as possible in the default textbox

My thought with the textbox is that it saves typing/time and ensures text consistency by simply having someone check one box or more.

Reply:
Right, I'm sold!